rmgpy/rmg/input.py

```diff
@@ -249,7 +249,15 @@ def simple_reactor(temperature,
         else:
             sensitive_species.append('all')
 
-    if not isinstance(T, list):
+    #Check the constant species exist
+    if constantSpecies is not None:
+        logging.debug('  Generation with constant species:')
+        for constantSpecie in constantSpecies:
```

Comment:
"specie" looks a little bit weird. Do you mind changing it to something else? Maybe like `const_spc`?